driver/driver.go

```diff
@@ -25,6 +25,8 @@ import (
 	"github.com/google/pprof/profile"
 )
 
+type HTTPServerArgs plugin.HTTPServerArgs
```

Comment:
The golint check in out test suite isn't passing right now because this exported type doesn't have a comment. 

Should `plugin.HTTPServerArgs` be defined as a separate type? I ask because I see both `plugin.HTTPServerArgs` and `HTTPServerArgs` used below.

Reply:
Thanks, added the comment.
Even if I define this as a separate type, I'll still have to convert to the `internal` variant below, so it wouldn't change anything there. I might be missing your point.